app/screens/bottom_sheet/index.tsx

```diff
@@ -191,10 +191,10 @@ const BottomSheet = ({
 
     if (isTablet) {
         return (
-            <>
+            <GestureHandlerRootView style={styles.container}>
```

Comment:
this should not be needed, can you check the function `withGestures` in `app/screens/index.tsx` to verify you have the latest main branch?

Reply:
Hi, I already just pulled the latest code.

I wonder if you refer to this code:
![Screen Shot 2023-08-08 at 19 44 54](https://github.com/mattermost/mattermost-mobile/assets/39702132/f4542d04-cc06-4f7e-9af0-066637d55df5)

I think the code lacks `withGestures` before `withSafeAreaInsets...`. What do you think if I add `withGestures` here for the bottom sheet, and remove the `GestureHandlerRootView` in this line of code?

Reply:
ok so the better fix is to wrap this with the `withGestures` function